### PR TITLE
Improvements to AIA+EUVI coordinates examples

### LINKED ISSUE #5533: Update Stereo Examples to have better plots
See https://github.com/sunpy/sunpy/pull/5529#pullrequestreview-758873316

We should fix these.

### PR DESCRIPTION
**Pull request by ayshih**
This PR fixes a number of issues I noted with coordinates examples that combine AIA and EUVI data (see https://github.com/sunpy/sunpy/pull/5529#pullrequestreview-758873316)

Closes #5533

### CODE REVIEW

**Comment by nabobalis** [comment]
I leave it you to you if you want to changelog it.

**Comment by ayshih** [comment]
I won't bother with a changelog here because #5529 didn't either

**Comment by ayshih** [comment]
Incidentally, the position of the features in the AIA map don't exactly line up to the features on the EUVI map.  There may be some issue with the metadata of one or both maps (observer location, plate scale, etc.).

**Comment by ayshih** [comment]
> Incidentally, the position of the features in the AIA map don't exactly line up to the features on the EUVI map. There may be some issue with the metadata of one or both maps (observer location, plate scale, etc.).

It shouldn't block this PR, but here's a figure showing what I mean (the contours are from AIA transformed into the EUVI coordinate frame):
download (36)

**Comment by ayshih** [comment]
The discrepancy is mostly due to the formation height of 304 A emission in the chromosphere, with the remaining discrepancy possibly due to inaccuracy in the reported pixel location of disk center in the EUVI metadata.

File: examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py [hunk 1]
```diff
@@ -42,6 +46,16 @@
                                       [-1100, 1100]*u.arcsec,
                                       frame=m.coordinate_frame))
         for m in sunpy.map.Map(files)}
+maps['AIA'].plot_settings['vmin'] = 0  # set the minimum plotted pixel value
+
+
+###############################################################################
+# We will be transforming coordinates where the formation height of 304 A
+# emission makes a difference, so we set the reference solar radius to 4 Mm
+# above the solar surface (see Alissandrakis 2019).
```
Comment (nabobalis) [inline examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py:55]: Do we want a url doi link for this?

Comment (ayshih) [reply examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py:55]: Probably a good idea.  https://doi.org/10.1007/s11207-019-1552-1 is the DOI URL.  Do we have a style for including such links in gallery examples?

Comment (nabobalis) [reply examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py:55]: I don't think we do but I would need to check. A standard inplace rst url link is good enough for me.

Comment (ayshih) [reply examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]: Added

Comment (nabobalis) [reply examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]: Thanks.

Comment (nabobalis) [reply examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]: Thanks.

**Comment by nabobalis** [comment]
I am not sure if this is an issue but:

https://sunpy--5534.org.readthedocs.build/en/5534/_images/sphx_glr_AIA_limb_STEREO_002.png
lacks a title? At least it is missing the EUVI 304 title from master.

**Comment by ayshih** [comment]
Ah, whoops, forgot to check that.  I pushed the title beyond the extent of the canvas, so I'll fix the figure size.

**Comment by Cadair** [comment]
I feel like this is a cool change and should at least have a changelog entry?

**Comment by ayshih** [comment]
> I feel like this is a cool change and should at least have a changelog entry?

Okay, added

**Review by nabobalis — APPROVED** [review]

File: examples/units_and_coordinates/AIA_limb_STEREO.py [hunk 2]
```diff
@@ -62,22 +63,28 @@
...
 
 fig = plt.figure()
 ax = plt.subplot(projection=maps['EUVI'])
 
 maps['EUVI'].plot()
 
-# Move the title so it does not clash with the extra labels.
-tx, ty = ax.title.get_position()
-ax.title.set_position([tx, ty + 0.08])
+# Crop the view
```
Comment (Cadair) [inline examples/units_and_coordinates/AIA_limb_STEREO.py:75]:
```suggestion
# Crop the view using pixel coordinates
```

**Pull request merged**